llm_bench/python/benchmark.py

```diff
@@ -674,6 +674,15 @@ def get_argprser():
         'if the value is False (default), input prompts are processed in interleave manner'
     )
     parser.add_argument('-od', '--output_dir', help='Save the input text and generated text, images to files')
+    parser.add_argument(
```

Comment:
Do we need to check the value of dynamic_quantization_group_size? Can this value be less than or equal to 0?

Reply:
This value can be equal to 0 which means disable dynamic quantizing.